build.gradle

```diff
@@ -385,6 +385,9 @@ subprojects {
             dependency "org.mockito:mockito-core:2.23.0"
             dependency "org.powermock:powermock-module-junit4:2.0.2"
             dependency "org.powermock:powermock-api-mockito2:2.0.2"
+            dependency "io.cloudevents:cloudevents-core:2.2.0"
+            dependency "org.junit.jupiter:junit-jupiter:5.7.0"
```

Comment:
To keep the UT code style consistent, it's better not add jupiter dependency in this pr.
And if you hope to add jupiter, it might be better to add another issue to discuss.